crates/primitives/src/state.rs

```diff
@@ -173,7 +173,7 @@ pub struct AccountInfo {
     /// code hash,
     pub code_hash: B256,
     /// code: if None, `code_by_hash` will be used to fetch it if code needs to be loaded from
-    /// inside of revm.
+    /// inside of REVM.
```

Comment:
```suggestion
    /// inside of `revm`.
```

I like it more as a small letters.